internal/client/client_test.go

```diff
@@ -1524,6 +1525,7 @@ var _ = Describe("client", func() {
 						svc.Name = configmapName
 						svc.Namespace = configmapNamespace
 						svc.Spec = corev1.ServiceSpec{
+							Type:     corev1.ServiceTypeLoadBalancer,
```

Comment:
Note: From k8s 1.29, a Service must have LoadBalancer as its spec.type to set status.loadBalancer.

> Setting the status.loadBalancer of a Service whose spec.type is not LoadBalancer was previously allowed, but any update to the metadata or spec would wipe that field. Setting this field is no longer permitted unless spec.type is LoadBalancer.

https://github.com/kubernetes/kubernetes/blob/e1e17973be8a023cda5f2b8c61a94bc32841acb5/CHANGELOG/CHANGELOG-1.29.md